root/etc/confd/templates/config.xml.tmpl

```diff
@@ -0,0 +1,41 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Nothing looks like it needs to be generated. This can likely be left out. The format does change for V2 though, but I can remove it from the PR.